superset/connectors/sqla/models.py

```diff
@@ -2099,6 +2099,25 @@ def load_database(self: SqlaTable) -> None:
     Column("rls_filter_id", Integer, ForeignKey("row_level_security_filters.id")),
 )
 
+RLSFilterGroups = DBTable(
+    "rls_filter_groups",
+    metadata,
+    Column("id", Integer, primary_key=True),
+    Column(
+        "group_id",
+        Integer,
+        ForeignKey("ab_group.id", name="rls_filter_groups_group_id_fkey"),
+    ),
+    Column(
+        "rls_filter_id",
+        Integer,
+        ForeignKey(
+            "row_level_security_filters.id", name="rls_filter_groups_rls_filter_id_fkey"
+        ),
+    ),
+    UniqueConstraint("group_id", "rls_filter_id", name="uq_rls_filter_group"),
+)
```

Comment:
for roles we have:
```
RLSFilterRoles = DBTable(
    "rls_filter_roles",
    metadata,
    Column("id", Integer, primary_key=True),
    Column("role_id", Integer, ForeignKey("ab_role.id"), nullable=False),
    Column("rls_filter_id", Integer, ForeignKey("row_level_security_filters.id")),
)
```
do we need to remove `nullable=False` now? because we can have a roles or a group?